frodo.js

```diff
@@ -1,5 +1,5 @@
-const evenNumbers = numbers.filter(num => num % 2 === 0);
```

Comment:
SCRIPT-The way you've refactored this complex method into simpler ones is great. It makes the code much more maintainable.